addons/website/static/src/snippets/s_masonry_block/001.scss

```diff
@@ -10,3 +10,7 @@
         --grid-item-padding-x: 0px;
     }
 }
+
+.s_masonry_block div.media_iframe_video {
+    width: 100%;
+}
```

Comment:
You forgot the versionning, it is for `s_masonry_block[data-vcss='001']` specifically :wink: 
But in fact, just put the rule above like this:
```scss
.s_masonry_block[data-vcss='001'] .row > div {
    display: flex;
    flex-direction: column;
    justify-content: center;

    div.media_iframe_video { // <-- your rule
      width: 100%;
    }
}
```
so it is grouped with the flex stuff and so we understand clearly that this is for the videos inside the flex columns :slightly_smiling_face: